vpd-tool/src/vpd_tool.cpp

```diff
@@ -1,5 +1,11 @@
 #include "vpd_tool.hpp"
 
+#include "tool_constants.hpp"
+#include "tool_types.hpp"
+#include "utils.hpp"
```

Comment:
why not tool_utils.hpp?

Reply:
renames files which are duplicate names in this repo.
utils.hpp is only file in this repo.

Reply:
For consistency, I would agree with @PriyangaRamasamy

Reply:
Updated